tests/io/fixtures/afp20.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="iso-8859-1"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess they won't change that for us, we need to use what we get from them

Reply:
then could you please give any idea how we handle it?

Reply:
I guess the library will handle it, so it might be ok actually. to test it you could save the file in that encoding from the header, seems like atm it's encoded as utf8